docs/docs/setting-up/networking-instructions/private-cluster.md

```diff
@@ -40,18 +52,21 @@ export BACALHAU_API_PORT=1234
 To connect another node to this private one, run the following command in your shell:
 
 ```
-bacalhau serve --private-internal-ipfs --peer /ip4/<ip-address>/tcp/1235/p2p/<peer-id> --ipfs-swarm-addr /ip4/<ip-address>/tcp/<port>/p2p/<peer-id>
+bacalhau serve \
+--private-internal-ipfs \
+--peer /ip4/<ip-address>/tcp/1235/p2p/<peer-id> \
+--ipfs-swarm-addr /ip4/<ip-address>/tcp/<port>/p2p/<peer-id>
 ```
 
 :::tip
-The exact command will be different on each computer and is outputted by the `bacalhau serve --node-type requester ...` command
+The exact command arguments will be different on each computer and is outputted by the `bacalhau serve --node-type requester ...` command
```

Comment:
```suggestion
The exact command arguments will be different on each computer and is output by the `bacalhau serve --node-type requester ...` command
```
I realise outputted was there before but it's the wrong tense.

Reply:
fixed